tools/testbench/topology.c

```diff
@@ -779,14 +784,24 @@ int parse_topology(struct tplg_context *ctx)
 
 		ctx->hdr = hdr;
 
-		if (hdr->index != ctx->pipeline_id) {
+		pipeline_match = false;
+		for (i = 0; i < tp->pipeline_num; i++) {
+			if (hdr->index == tp->pipelines[i]) {
+				pipeline_match = true;
+				break;
+			}
+		}
+
+		//pipeline_match = true; // Hack to get AEC reference connect included
```

Comment:
Wonder if need for mode where everything in topology is tried to be instantiated, not just -p x,y,z. Remove //.